.github/workflows/integration-tests-benchmarks.yml

```diff
@@ -1,9 +1,18 @@
 name: "Integration Tests - Benchmarks"
 on:
+  schedule:
```

Comment:
Just thinking if it's necessary, because they will anyway run on every merge/push to `main`? If there were no changes on main and a cron job kicks in, it's still useless probably, because there were nothing new on main?

Reply:
you may be right on this
let's remove pushing to main: we may have multiple pushes to main daily, so we could end up with a lot of tests running on saucelabs, risking timeouts and slowdowns. Also, we would still run the single benchmark for every pr, so running only nightly is enough at the moment. We can still change it later

Reply:
Yeah, sounds good 👍 just need some kind of reminder on slack or somewhere to check the benchmarks from the nightly builds

Reply:
We don't need to do it now, but would be cool to either post on slack or automatically create an issue if there were regressions after the nightly build

Reply:
actually I guess it's pretty straightforward - if a test run failed, we just use some action to create an issue and that's it :D most important is that there shouldn't be flaky tests to not overwhelm us with issues haha